src/modelfree/train.py

```diff
@@ -209,11 +212,12 @@ def load_default(env_name, config_dir):
 
 @train_ex.config
 def wrappers_config(env_name):
+    victim_noise = False  # enable adding noise to victim
```

Comment:
why did we reorder this? It's not a big deal but by default would keep things alphabetically ordered.